core/src/main/java/org/springframework/security/authentication/UserDetailsRepositoryReactiveAuthenticationManager.java

```diff
@@ -43,6 +44,8 @@
 
 	private Scheduler scheduler = Schedulers.parallel();
 
+	private UserDetailsChecker userDetailsChecker;
```

Comment:
We should default this to just do nothing, so we don't need to worry about null values.

Reply:
Do you mean build some kind of `NoOpUserDetailsChecker` and default to that? Or do something like this inside `UserDetailsRepositoryReactiveAuthenticationManager`

```java
private UserDetailsChecker userDetailsChecker = userDetails -> {};
```

Reply:
Just use a lambda